rgw/v2/tests/s3_swift/resuables/sts.py

```diff
@@ -0,0 +1,71 @@
+import os
+import sys
+
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../../..")))
+import json
+import logging
+import time
+
+import v2.utils.utils as utils
+from v2.lib.exceptions import TestExecError
+from v2.lib.rgw_config_opts import ConfigOpts
+from v2.lib.s3.auth import Auth
+
+log = logging.getLogger()
+TEST_DATA_PATH = None
+
+
+def add_sts_config_to_ceph_conf(
+    ceph_config_set, rgw_service, sesison_encryption_token="abcdefghijklmnoq"
+):
```

Comment:
Can we have docStrings for the new methods?

Reply:
@psathyan doc strings added